packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -34,57 +32,33 @@ export const pageQuery = graphql`
 
 <p>{props.pageContext.frontmatter.description}</p>
 
-<Box marginTop="space100" marginBottom="space100">
-  <StatusLegend />
-</Box>
-
 <Box>
 
-## Release 2020.02.25
-
-Release date: February 25, 2020
+  ## Upcoming Release
 
-**Design releases**
+  Paste's upcoming release will be on **April 7, 2020**. This is what we've planned:
 
-- Design more form field components (Select, Checkbox and Checkbox group, and Radio and Radio Group). 📝
-- Design a Form composition component. 📝
-- Design a Toast component. 🍞
+  - Form components are the bread and butter for most Twilio experiences. With this release we'll ship seven components (Text Input, Textarea, Select, Checkbox, Checkbox Group, Radio, and Radio Group) for Sketch and React. 📝 
 
-**Engineering releases**
+  - We'll ship a Content guide for Twilio UX, which should help you to feel more confident in writing concise, clear, and consistent product content. 🖊 
 
-- Publish the Modal Primitive component
-- Publish a Grid layout component
+  - Modals, an important container element to focus user attention, will be shipped for Sketch and React. 🔳
 
-**Other**
-
-- Add an icon list to the [icon system guidelines](/icon-system) with more design guidelines. Right now, you can access our <Anchor href="https://twilio-labs.github.io/paste/?path=/story/overview-icons--list"> full icon list on Storybook</Anchor>.
-
----
+  ---
 
-## Release 2020.04.07
+  ## Future Releases
 
-Release date: April 7, 2020
+  Future releases of Paste are scheduled for every six weeks, with the next release on **May 19, 2020**. Our guidance on what's planned will update as we get closer to the date. For now, here's what we're hoping to do:
 
-**Design releases**
+  - Ship a Form composition component. 📝
```

Comment:
Ship three more form components (Checkbox, Checkbox Group, and Radio Group) for Sketch and React." 📝